src/rules/order.js

```diff
@@ -197,8 +197,11 @@ function fixOutOfOrder(context, firstNode, secondNode, order) {
     newCode = newCode + '\n'
   }
 
-  const message = '`' + secondNode.name + '` import should occur ' + order +
-      ' import of `' + firstNode.name + '`'
+  const nameResolver = (importEntry) => importEntry.type === 'import:object' ?
+    context.getSourceCode().getText(importEntry.node.moduleReference) : importEntry.name
+
+  const message = '`' + nameResolver(secondNode) + '` import should occur ' + order +
+    ' import of `' + nameResolver(firstNode) + '`'
```

Comment:
```suggestion
  const message = `\`${nameResolver(secondNode)}\` import should occur ${order} import of \`${nameResolver(firstNode))\``
```

Reply:
Will be fixed after applying the change mentioned here:
https://github.com/benmosher/eslint-plugin-import/pull/1831#discussion_r443265731